androidApp/src/main/java/io/spherelabs/anypass/android/InAppUpdate.kt

```diff
@@ -0,0 +1,58 @@
+package io.spherelabs.anypass.android
+
+import android.app.Activity
+import androidx.activity.ComponentActivity
+import androidx.activity.result.contract.ActivityResultContracts
+import androidx.fragment.app.FragmentActivity
+import com.google.android.play.core.appupdate.AppUpdateManagerFactory
+import com.google.android.play.core.appupdate.AppUpdateOptions
+import com.google.android.play.core.install.model.AppUpdateType
+import com.google.android.play.core.install.model.UpdateAvailability
+
+class InAppUpdate(
+    activity: FragmentActivity,
+    private val onUpdateFailed: () -> Unit
+) {
+
+    private val appUpdateManager = AppUpdateManagerFactory.create(activity)
```

Comment:
```
private val appUpdateManager by lazy {
            AppUpdateManagerFactory.create(activity)
}
```

I would change it this way.

Reply:
https://developer.android.com/guide/playcore/in-app-updates/kotlin-java#kotlin

Reply:
Yeah that's right thank you. I amend changes into last commit, so you might like to check it out.